packages/frontend/tests/integration/components/pod/chat/chat-entry/emoji-test.ts

```diff
@@ -0,0 +1,70 @@
+import { module, test } from 'qunit';
+import { render, typeIn } from '@ember/test-helpers';
+import { setupRenderingTest } from 'ember-qunit';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | chat-entry', function(hooks) {
+  setupRenderingTest(hooks);
+
+  module('emoji code replacement', function() {
+    module('there are no emoji codes to replace', function() {
+      test('result does not contain emoji', async function(assert) {
+        assert.expect(2);
+        await render(hbs`<Pod::Chat::ChatEntry />`);
```

Comment:
Since both of these tests share the same first 4 lines, how do you feel about the render, and empty check/assert getting pulled up into a before each?

Reduce duplication, and results in what you're actually testing being more focussed